### PR TITLE
Ete testcase add more models

### PR DESCRIPTION
**Pull request by zhulinJulia24**
add more models into pytorch chat and pipeline
fix workflow to support diff tokenizer version and stable torch version
fix temperature value
fix pipeline for multiple chat

### CODE REVIEW

File: autotest/test_order3_pipeline_chat.py [hunk 1]
```diff
@@ -13,86 +13,112 @@ class TestPipelineChat:
...
     @allure.story('Baichuan2-7B-Chat-inner-w4')
-    def test_chat_Baichuan2_7B_Chat_inner_w4(self, config, common_case_config):
+    def test_pipeline_chat_Baichuan2_7B_Chat_inner_w4(self, config,
+                                                      common_case_config):
         run_pipeline_chat_test(config, common_case_config,
                                'Baichuan2-7B-Chat-inner-w4')
 
+    @pytest.mark.Baichuan2_13B_Chat
+    @allure.story('Baichuan2-13B-Chat')
+    def futurn_test_pipeline_chat_Baichuan2_13B_Chat(self, config,
```
Comment (RunningLeon) [inline autotest/test_order3_pipeline_chat.py:100]: what's `futurn`

File: .github/workflows/daily_ete_test.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ env:
 jobs:
   test_functions:
     runs-on: [self-hosted, linux-a100]
-    timeout-minutes: 120 # 2hours
+    timeout-minutes: 180 # 2hours
```
Comment (RunningLeon) [inline .github/workflows/daily_ete_test.yml:16]: do you have a successfully run log?

File: autotest/test_order3_pipeline_chat.py (hunk 1 above)
Comment (zhulinJulia24) [reply autotest/test_order3_pipeline_chat.py:100]: 之前搞错了，添加了baichuan2_13b_chat的用例，执行异常。和晗姐确认baichuan2_13b_chat turbomind还不支持，先加torch的。不想删除了，就先添加future....等后续支持了删除即可

futurn是个typo，我改一下

File: autotest/chat_prompt_case.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ identity_case:
     - 你好，你叫什么名字#hi, what's your name:
 
 memory_test:
-    - 介绍成都的景点#please introduce some attractions in Chengdu:
+    - 介绍成都的3个景点#please introduce tree attractions in Chengdu:
```
Comment (RunningLeon) [inline autotest/chat_prompt_case.yaml:5]: what's tree attractions?

Comment (RunningLeon) [reply autotest/chat_prompt_case.yaml:5]: is it `three`?

Comment (zhulinJulia24) [reply autotest/chat_prompt_case.yaml:5]: 不是，只是想通过数量减少输出长度。memorytest晗姐给了别的建议增加稳定性，我在实验，还得要一会

Comment (zhulinJulia24) [reply autotest/chat_prompt_case.yaml]: https://github.com/zhulinJulia24/lmdeploy/actions/runs/7767525206

File: .github/workflows/daily_ete_test.yml [hunk 4]
```diff
@@ -105,7 +103,7 @@ jobs:
           pytest autotest -m '(Baichuan2_7B_Chat or Baichuan2_13B_Chat) and not pipeline_chat_pytorch' --alluredir=allure-results
       - name: Test lmdeploy - rerun fail cases
         run: |
-          pytest autotest -m '(Baichuan2_7B_Chat or Baichuan2_13B_Chat) and not pipeline_chat_pytorch' --alluredir=allure-results --lf
+          pytest autotest --alluredir=allure-results --lf
```
Comment (RunningLeon) [inline .github/workflows/daily_ete_test.yml:101]: may remove line 112 since there 's no `ALLURE_DIR`

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**